lib/Data/Content/CreateContentDraftData.php

```diff
@@ -0,0 +1,19 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace EzSystems\RepositoryForms\Data\Content;
+
+use eZ\Publish\API\Repository\Values\ValueObject;
+
+class CreateContentDraftData extends ValueObject
+{
+    public $contentId;
```

Comment:
Please disregard this. It is valid for values that are set in the repository. This is input data, meant to be manipulated by the client (the mapper here).